src/ui/assets/AssetsList.js

```diff
@@ -197,15 +200,23 @@ class AssetsListBody extends Component {
       onApplyFilteredSearch,
       onChangeAssetsView,
       onApplySort,
-      paginationOptions,
       activeFilters,
+      lastPage,
+      totalItems,
+      pageSize: perPage,
+      page,
+      perPageOptions,
+      apiUrl,
     } = this.props;
+    const pagination = {
+      page,
+      perPage,
+      perPageOptions,
+    };
     const { mobile } = this.state;
-    const {
-      lastPage, totalItems, perPage, page,
-    } = paginationOptions;
     const itemsStart = totalItems === 0 ? 0 : (page - 1) * perPage + 1;
     const itemsEnd = Math.min(page * perPage, totalItems);
+    const apiDomain = apiUrl.match(/:\/\/(.[^/]+)/)[1];
```

Comment:
I suggest you use the native web api `URL` (see here for specs https://developer.mozilla.org/en-US/docs/Web/API/URL) and just use the method `origin` to get the domain with protocol.